master_handler.py

```
@@ -143,46 +148,61 @@ def validate_and_save_scrapers():
-                    "function_name": function_name,
-                    "status": "success",
-                    "message": "비동기 호출 성공",
-                }
-            )
-
+            print(f"   ✅ {function_name} 호출 완료")
+            return {
+                "function_name": function_name,
+                "status": "success",
+                "message": "호출 성공",
+            }
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)